### ISSUE TITLE
Mind providing some example screenshots?

### ISSUE THREAD

**Issue by carstenhag** [HiFIi/ToolbarMenudrawer-2.0#1]
:)

**Comment by HiFIi** [comment]
@Moter8 sorry it took me so long to reply.

By 'Example Screenshots' , do you mean examples of the various sections of the app?

**Comment by carstenhag** [comment]
Yeah, how the Toolbar/Drawer looks, it's animations stuff like that!

**Comment by HiFIi** [comment]
Yeah, sure thing dude. I'll get around to that as soon as possible.

Thank you!

**Comment by HiFIi** [comment]
Reopened this because I haven't completed it yet
